### PR TITLE
Readme polishing

### PR DESCRIPTION
**Pull request by khawkins98**
New to this project and here's a few things I came across as I read the intro:

- Removed the link to https://trunkmaster.github.io/nextspace/ as the content there seemed from a stale version, and there's no longer a `gh-pages` branch?
- Added some information on installing as I had to guess to go to the "releases" to find out what to do
- Made it clearer that this is a CentOS 7-based project (good to know)
- A few general text tweaks that I think make reading things easier

Some subjective edits, hopefully they're helpful.

p.s. Neat project! (Still waiting on CentOS to install)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,26 +46,27 @@ A fast and elegant Workspace Manager - using multithreading to provide maximum s
...
-> * WindowMaker with some good configuration defaults only
-> * Another implementation of WindowMaker.
 
-> It's written from scratch. Some WindowMaker code is a part of Workspace (as well as configuration defaults) to provide window management functions. It's tightly coupled with Workspace to provide seamless intergation. Configurable parameters of the integrated WindowMaker are spread across Workspace's Preferences and Preferences application.
+Note: Workspace is NOT:
+* WindowMaker with some patches
+* WindowMaker with some good configuration defaults only
+* Another implementation of WindowMaker.
+
+Workspace is written from scratch. Some WindowMaker code is a part of Workspace (as well as configuration defaults) to provide window management functions. The code is tightly coupled with Workspace to provide seamless intergation. Configurable parameters of the integrated WindowMaker are spread across Workspace's Preferences and Preferences application.
```
Comment (trunkmaster) [inline README.md]: I'd like to keep it as side note. Replace `*` with `>` please.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,42 @@
 # NEXTSPACE
 
-NEXTSPACE is a desktop environment that brings a [NeXTSTEP](https://en.wikipedia.org/wiki/NeXTSTEP) look and feel to Linux. I try to keep the user experience as close as possible to the original NeXT's OS. It is developed according to the ["OpenStep User Interface Guidelines"](http://www.gnustep.org/resources/documentation/OpenStepUserInterfaceGuidelines.pdf).
-> An explanation of the NeXTSTEP / NextStep naming convention is discussed in further detail [here](Documentation/OpenStep%20Confusion.md).
+NEXTSPACE is a desktop environment that brings a [NeXTSTEP](https://en.wikipedia.org/wiki/NeXTSTEP) look and feel to CentOS 7 Linux. I try to keep the user experience as close as possible to the original NeXT OS. It is developed according to the ["OpenStep User Interface Guidelines"](http://www.gnustep.org/resources/documentation/OpenStepUserInterfaceGuidelines.pdf).
```
Comment (trunkmaster) [inline README.md]: I develop and test my code in CentOS. Although I see some demand for Ubuntu packages. I think we can leave just "Linux" here.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,42 @@
...
+
+* An explanation of the NeXTSTEP / NextStep naming convention is discussed in further detail [here](Documentation/OpenStep%20Confusion.md).
```
Comment (trunkmaster) [inline README.md]: I'd like to keep it as side note. Replace `*` with `>` please.

**Comment by khawkins98** [comment]
Updated that Cent OS reference and the `>`s .

I think those fall well into the category of my subjective edits — I found them confusing as first I read them as blockquotes. Ideally the `<aside>` element would be supported by GitHub Markdown. 

Maybe _italics_ would do the job. But it's minor stuff.

**Comment by trunkmaster** [comment]
Thanks. I've blockquoted additional, not so important information. Merging.

**Pull request merged**